docs/UserGuide.adoc

```diff
@@ -96,14 +96,14 @@ This command takes no arguments.
 ==== Adding a person: `add`
 
 Adds a person to the address book. +
-Format: `add n/NAME [p/PHONE_NUMBER]... [e/EMAIL]... [a/ADDRESS]... [fb/FACEBOOK] [tw/TWITTER] [ig/INSTAGRAM] [t/TAG]...` +
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [fb/FACEBOOK] [tw/TWITTER] [ig/INSTAGRAM] [t/TAG]...` +
 Alias: `a`
 
 [TIP]
-A person can have any number of tags, phone numbers, emails or addresses (including 0) (Coming v2.0)
+A person can have any number of tags, phone numbers, emails or addresses (including 0) (Not implemented yet. Coming v2.0)
```

Comment:
Let's change this to `Coming in v2.0` to make it consistent with the below as well

Reply:
I changed it to make it more explicit what features haven't been included yet. I think I changed it all throughout? Unless I missed something.